common/Configuration.js

```diff
@@ -21,6 +21,7 @@ module.exports = (function(){
     if (fs.existsSync(mappingFilePath))
     {
         var mappingContent = fs.readFileSync(mappingFilePath, 'utf8');
+        mappingContent= mappingContent.replace(/~/g,hostname.homedir());
```

Comment:
maybe it's better to move that ourside the scope  and remove the one line line#16??